.github/workflows/ci.yaml

```diff
@@ -0,0 +1,96 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  cargo-test:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macos-10.15]
+        include:
+          - os: ubuntu-18.04
+            features: --features "blocking beta"
```

Comment:
what are these features vs the ones spelled out in the  uses: actions-rs/cargo@v1 block below?

Reply:
The block you mention is in a different job, so unrelated. But I'll be updating our matrix to cover features better in my next commit.